languages/elixir/exercises/concept/enum/.docs/introduction.md

```diff
@@ -0,0 +1,58 @@
+`Enum` is a very useful module that provides a set of algorithms for working enumerables. It offers sorting, filtering, grouping, counting, searching, finding min/max values, and much more.
+
+In Elixir, an enumerable is any data type that implements the `Enumerable` protocol. The most common of those are lists and maps.
+
+```elixir
+Enum.sort([4, 2, 3, 1])
+# => [1, 2, 3, 4]
+```
+
+```elixir
+Enum.empty?(%{})
+# => true
+```
+
+Many `Enum` functions accept a function as a second argument.
+
+```elixir
+Enum.filter([1, 2, 3, 4, 5], fn x -> x > 3 end)
+# => [4, 5]
+```
+
+The most common `Enum` functions and `map` and `reduce`.
+
+`Enum.map/2` allows you to replace every element in an enumerable with another element. The second argument to `Enum.map/2` is a function that accepts the original element and returns its replacement.
+
+```elixir
+Enum.map([1, 2, 3, 4, 5], fn x ->
+  x * x
+end)
+# => [1, 4, 9, 16, 25]
+```
+
+`Enum.reduce/2` allows you to _reduce_ the whole enumerable to a single value. To achieve this, a special variable called the _accumulator_ is used. The accumulator carries the intermediate state of the reduction between iterations.
```

Comment:
In some languages this is called a _fold_, which may allow you to refine it without reusing the word reduce.

Reply:
I also know the name _inject_ from Ruby, but _reduce_ is my favorite because it pictures well that you're left with something potentially much smaller and not resembling the input. _Fold_ makes me think of cards and origami... but I am aware it exists in Elixir as `List.foldr` and `List.foldl`.

Does this repetition sound bad to a native ear?

Reply:
I wouldn't say it sounds bad, but I think when defining a word it's always a bit more clear to use a different word to describe it.

Now that I re read it with the italics, I can see what you are going for here, and it's good. :)